terraform/script.py

```diff
@@ -63,28 +59,76 @@ def get_envs():
     nodes_number = os.getenv("NODES_NUMBER")
     print(cluster_name)
 
-def confirm_config():
-    True
+
+def select_workspace():
+    global kube
+    if os.getenv("TF_VAR_CLOUDPROVIDER") == "aws":
+        kube = str("eks")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "gcp":
+        kube = str("gke")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "az":
+        kube = str("azkube")
+    
+    os.environ["TF_VAR_KUBE"] = kube
```

Comment:
WDYT about renaming this `kube` variable to something else, for me it's hard to understand the purpose of this variable from its name. Maybe something like `kubernetes_provider`?

Reply:
OK, changed.